web/app/scripts/views/record/list-controller.js

```diff
@@ -33,12 +37,27 @@
                 });
         }
 
-        function loadRecords() {
+        /*
+         * Loads a page of records from the API
+         * @param {int} offset Optional offset value, relative to current offset, used
+         *                     for pulling paginated results. May be positive or negative.
+         * @return {promise} Promise to load records
+         */
+        function loadRecords(offset) {
             /* jshint camelcase: false */
-            return Records.get({ record_type: $stateParams.rtuuid })
+            var params = { record_type: $stateParams.rtuuid };
             /* jshint camelcase: true */
+
+            if (offset) {
+                ctl.currentOffset += offset;
+                if (ctl.currentOffset) {
+                    params.offset = ctl.currentOffset;
+                }
+            }
+
+            return Records.get(params)
```

Comment:
It seems a bit strange to me that the Angular $resource is causing us to avoid using the API next/previous URLs here. I'm not sure there's anything we can do about it, but I wanted to bring it up, maybe there's a solution I don't know about.